docs/src/testing/storage-cheatcodes.md

```diff
@@ -0,0 +1,164 @@
+# Direct Storage Access
+
+In some instances, it's not possible for contracts to expose API that we'd like to use in order to initialize 
+the contracts before running some tests. For those cases `snforge` exposes storage-related cheatcodes,
+which allow manipulating the storage directly (reading and writing).
+
+In order to obtain the variable address that you'd like to write to, or read from, you need to use either:
+- `selector!` macro - if the variable is not a mapping
+- `map_entry_address` function in tandem with `selector!` - for key-value pair of a map variable
+
+## Example: Felt-only storage
+This example uses only felts for simplicity
+
+```rust
+#[starknet::contract]
+mod Contract {
+    #[storage]
+    struct Storage {
+        plain_felt: felt252,
+        mapping: LegacyMap<felt252, felt252>,
+    }
+}
+
+// ...
+use snforge_std::{ store, load, map_entry_address };
+
+#[test]
+fn store_and_load_with_plain_felt() {
+    // ...
+    store(contract_address, selector!("plain_felt"), array![123].span());
+    // plain_felt = 123
+    let loaded = load(contract_address, selector!("plain_felt"), 1);
+    assert(loaded.len() == 1, 'Wrong loaded vector');
+    assert(*loaded.at(0) == 123, 'Wrong loaded value');
+}
+
+
+#[test]
+fn store_and_load_map_entry() {
+    // ...
+    store(
+        contract_address, 
+        map_entry_address(
+            selector!("mapping"), // Providing variable name
+            array![123].span(),   // Providing mapping key 
+        ),
+        array![321].span()
+    );
+    
+    // mapping = { 123: 321, ... }
+    let loaded = load(
+        contract_address, 
+        map_entry_address(
+            selector!("mapping"), // Providing variable name
+            array![123].span(),   // Providing mapping key 
+        ),
+        1,
+    );
+    
+    assert(loaded.len() == 1, 'Expected 1 felt loaded');
+    assert(*loaded.at(0) == 321, 'Expected 321 value loaded');
+}
+```
+
+## Example: Complex structures in storage
+This example uses a complex key and value, with `serde` as default serialization method.
```

Comment:
```suggestion
This example uses a complex key and value, with `Serde` as default serialization method.
```

Reply:
We should rewrite it so that the user knows that deriving `Store` makes the type be stored as if it would be serialised with `Serde`, this is quite confusing